addons/mail/i18n/mail.pot

```diff
@@ -500,6 +500,14 @@ msgid ""
 "be transformed into \"Good Morning\"."
 msgstr ""
 
+#. module: mail
+#: code:addons/mail/models/res_partner.py:0
+#, python-format
+msgid ""
+"%(email)s is not recognized as a valid email. This is required to create a "
```

Comment:
Being that annoying guy ... now that it begins with a "%", is it at its right place in the pot file :D ?